src/components/footers/Footer2.tsx

```diff
@@ -0,0 +1,20 @@
+import Image from 'next/image'
+import Link from 'next/link'
+import React from 'react'
+
+export default function Footer2() {
+  return (
+    <div className='fixed flex justify-center items-center bottom-0 w-full border-t-2 border-[#A135E8]' style={{height: '70px'}}>
```

Comment:
It is possible to use custom sizes in tailwind, it is not necessary to use the 'style' tag, it would be "h-[70px]"